infra/lib/env-config.ts

```diff
@@ -23,6 +23,10 @@ export type EnvConfig = {
   usageReportUrl?: string;
   fhirServerUrl?: string;
   systemRootOID: string;
+  cwMemberName: string;
+  cwMemberOID: string;
+  cwGatewayEndpoint: string;
+  cwGatewayAuthorizationServerEndpoint: string;
```

Comment:
We could have these under a `commonwell` prop or something, so we don't have to import them individually on the stack code, like we did for the `slack` config.

Reply:
Thank @Goncharo, he suggested that on the PR we introduced the slack config. 😄